org.sonarlint.eclipse.core/src/org/sonarlint/eclipse/core/internal/jobs/ServerUpdateJob.java

```diff
@@ -49,6 +50,9 @@ protected IStatus run(IProgressMonitor monitor) {
     try {
       server.updateStorage(monitor);
     } catch (Exception e) {
+      if (e instanceof CanceledException && monitor.isCanceled()) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Replace the usage of the "instanceof" operator by a catch block. ([squid:S1193](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1193))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarlint.eclipse%3Asonarlint-eclipse-parent&pullRequest=158&issues=AWOCPl0-LjbbHpSefg01&open=AWOCPl0-LjbbHpSefg01)

[ISSUE_KEY]: AWOCPl0-LjbbHpSefg01